html2asketch/symbolMaster.js

```diff
@@ -77,7 +77,7 @@ class SymbolMaster extends Base {
     obj.includeBackgroundColorInExport = true;
     obj.resizesContent = false;
     obj.includeBackgroundColorInInstance = false;
-    obj.symbolID = generateID();
+    obj.symbolID = this._name;
```

Comment:
Just a heads up, this will conflict with #63

Reply:
it probably will. although im not sure what to do with it